<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuScreen</class>
 <widget class="QDialog" name="MenuScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Competitors</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_10">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QComboBox" name="competitorOneBox">
              <property name="maxVisibleItems">
               <number>11</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="competitorTwoBox"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_6">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Leg Start Value</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QComboBox" name="legStartValueBox">
           <item>
            <property name="text">
             <string>301</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>501</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>801</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Number of Legs</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QSpinBox" name="numberOfLegsSpinBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Number of Matches</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QSpinBox" name="numberOfMatchesSpinBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_7">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="dateTimeMenu"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Official</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="officialSpinBox">
           <item>
            <property name="text">
             <string>ref 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ref 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ref 3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ref 4</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_9">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Location</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="locationPickBox">
           <item>
            <property name="text">
             <string>Charger Union</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Library</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tech Hall</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Charger Park</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="menuBeginButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(114, 131, 255);</string>
        </property>
        <property name="text">
         <string>BEGIN</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
